drizzlepac/haputils/cell_utils.py

```diff
@@ -372,6 +374,31 @@ def build(self, expnames, scale=False, scale_kw='EXPTIME'):
         # Compute the bounded WCS for this mask of exposed pixels
         self.find_bounded_wcs()
 
+    def extract_mask(self, filename):
+        """Extract a total_mask from the SCI data directly"""
+        sciext = ("SCI", 1)
+        # Try to use the WHT array as a msk
```

Comment:
Correct comment regarding use of WHT array.

Reply:
Not only did I fix this comment in 8592ee9, I updated the logic to work with simple FITS files as well, since DRZ files could be simple when 'build=no' in astrodrizzle.